### ISSUE TITLE
Night mode should be bound to alternate tagged stylesheet

### ISSUE THREAD

**Issue by danielweck** [readium/readium-js-viewer#111]
http://www.idpf.org/epub/altss-tags/#app-night-version-style-set-tags
